services/BasicTypes.proto

```diff
@@ -155,7 +155,7 @@ message ThresholdKey {
     KeyList keys = 2; // List of all the keys that can sign
 }
 
-/* A list of keys */
+/* A list of keys that requires all keys (M-of-M) to sign unless otherwise specified in documentation. A KeyList may contain repeated keys but all repeated keys are only required to sign once. */
```

Comment:
A KeyList may contain repeated keys, but repeated keys are only required to sign once.